lib/workers/repository/config-migration/branch/commit-message.ts

```diff
@@ -4,43 +4,61 @@ import type { CommitMessage } from '../../model/commit-message';
 import { CommitMessageFactory } from '../../model/commit-message-factory';
 
 export class ConfigMigrationCommitMessageFactory {
-  private readonly config: RenovateConfig;
+  private commitMessage: string | null = null;
+  private prTitle: string | null = null;
 
-  private readonly configFile: string;
+  constructor(
+    private readonly config: RenovateConfig,
+    private readonly configFile: string
+  ) {}
 
-  constructor(config: RenovateConfig, configFile: string) {
-    this.config = config;
-    this.configFile = configFile;
-  }
-
-  create(): CommitMessage {
+  private create(isTitle = false): CommitMessage {
+    const messageTopic = isTitle
+      ? `Migrate renovate config`
+      : `Migrate config ${this.configFile}`;
     const { commitMessage } = this.config;
+    let { commitMessageAction, commitMessageTopic } = this.config;
 
-    this.config.commitMessageAction =
-      this.config.commitMessageAction === 'Update'
-        ? ''
-        : this.config.commitMessageAction;
+    commitMessageAction =
+      commitMessageAction === 'Update' ? '' : commitMessageAction;
 
-    this.config.commitMessageTopic =
-      this.config.commitMessageTopic === 'dependency {{depName}}'
-        ? `Migrate config ${this.configFile}`
-        : this.config.commitMessageTopic;
+    commitMessageTopic =
+      commitMessageTopic === 'dependency {{depName}}'
+        ? messageTopic
+        : commitMessageTopic;
```

Comment:
These i think shouldn't be user configurable in the config migration case.
wdyt?